create-creds

```diff
@@ -4,12 +4,12 @@ import sys
 from helpers import generate_credential_tokens, require_secrets
 
 
-def main(creds_in, creds_out):
-    require_secrets('osftok', 'triggered-by')
-    generate_credential_tokens(creds_in, creds_out)
+def main(creds_out):
+    require_secrets('osftok', 'triggered-by', 'targets.sec')
```

Comment:
I suppose `root.json` and `targets.pub` are also required secrets.

Reply:
good catch. i'll make that change and just push to master to clean this up.